sample/general/jobs/sample_pytest.py

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
let's probably make this a real test case and a real sample component? So we could use this test case as an indicator that our infra is running successfully.

Reply:
done